python/cloud-run-python-hello-world/cloudbuild.yaml

```diff
@@ -0,0 +1,30 @@
+steps:
+- id: docker build
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/${_PROJECT_ID}/${_SERVICE}', '.']
+
+- id: docker push
+  name: 'gcr.io/cloud-builders/docker'
+  args: ['push', 'gcr.io/${_PROJECT_ID}/${_SERVICE}']
+
+- id: Deploy to Cloud Run
+  name: 'gcr.io/cloud-builders/gcloud'
+  args:
+  - beta
```

Comment:
is it still beta?
I was able to deploy using **gcloud run deploy**

Reply:
Done in next commit.